FinancialTracker/ViewControllers/HomeViewController.swift

```diff
@@ -6,16 +6,67 @@
 //
 
 import UIKit
+import Charts
 
 class HomeViewController: UIViewController {
-
+    @IBOutlet var expenseChart: PieChartView!
+    
     override func viewDidLoad() {
         super.viewDidLoad()
 
         self.title = "Home"
         self.navigationItem.setHidesBackButton(true, animated: true)
         self.navigationItem.rightBarButtonItem = UIBarButtonItem(title: "Sign Out", style: .plain, target: self, action: #selector(signOut))
     }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        
+        customizeChart(data: FirebaseHandler.shared.currentUser?.expenses)
+    }
+    
+    func customizeChart(data: [Expense]?) {
```

Comment:
`private func`?

Reply:
Also, make the function accept non-optional `[Expense]`, and use ternary operator `... ?? []` when you use the function